content/community/articles.md

```diff
@@ -2,52 +2,35 @@
-*   [Getting started with Apache Camel video from BarcelonaJUG (January 2014)](http://www.youtube.com/watch?v=lOxUs_JleSs&feature=youtu.be&a) by Claus Ibsen
-*   [Camel in the cloud demo from CamelOne 2013](http://vimeo.com/68442425) - Video presentation from CamelOne 2013 demonstrating Camel in the cloud using [Fuse Fabric](https://fabric8.io/) and [hawtio](http://hawt.io/)
-*   [Apache Camel Essential Components](https://blog.christianposta.com/webinar/camel-essential-components-webinar/) _by Christian Posta_ (January 2013)
-*   [Leverage Enterprise Integration Patterns with Apache Camel and Twitter](https://mnclip.info/w8WtDnZz8ac-leverage-eips-with-apache-camel-and-twitter.html) _by Bruno Borges_ (JavaOne 2012)
-*   [Riding the Apache Camel into the cloud](http://www.infoq.com/presentations/Riding-Apache-Camel-Cloud) _James Strachan_ (QCon 2012)
-*   [Migrating akka-camel module to Akka 2.x](http://skillsmatter.com/podcast/scala/akka-2-x) _by Peter Gabryanczy_ (ScalaDays 2012)
-*   [Confident Data Transfers with Apache Camel Security](http://www.youtube.com/watch?v=YiG5_rGLapU) _by Dominik Schadow_ (JavaOne 2012)
-*   [CamelOne 2012 Videos](http://camelone.org/camelone-2012/) - All the videos from the [CamelOne 2012 conference](http://fusesource.com/apache-camel-conference-2012) (May 2012)
-*   [JavaOne 2012 - Leverage Enterprise Integration Patterns with Apache Camel and Twitter](http://java.dzone.com/articles/leverage-enterprise) _by Bruno Borges_ (JavaOne 2012)
-*   [Using Apache Camel and Java EE in an OSGi World](http://www.java-tv.com/2012/02/28/using-apache-camel-and-java-ee-in-an-osgi-world/) _by Jeff Genender and Johan Edstrom_ (JavaOne 2011)
-*   [Er Apache Camel riktig valg for deg? Lytt til erfarne Camel spotters](http://vimeo.com/28760446)  _by Rune Peter Bjørnstad, and Bjørn Nordlund_(September 2011)
-*   [In the brain of James Strachan, riding the Camel](http://skillsmatter.com/podcast/design-architecture/riding-camel/JS-2327) - _by_ _[James Strachan](http://macstrac.blogspot.com/)_ (August 2011)
-*   [CamelOne 2011 conference](http://camelone.org/camelone-2012/) (May 2011)
-*   [What Riding the Camel can make integration easier for you](http://vimeo.com/28760458) _by_ _[Claus Ibsen](http://davsclaus.blogspot.com)_ (September 2011)
-*   [Un plongeon dans les frameworks d'intégration Java: Apache Camel et Spring Integration](https://www.youtube.com/watch?v=gkUzmbwJr8k) _by Guillaume Giamarchi and Grégory Boissinot_ (October 2013)
-*   [JEEConf 2011 - What riding the Camel can do to make integration easier for you](http://jeeconf.com/program/camel/) _by Claus Ibsen_ (May 2011).
-*   [Introduction to messaging with ActiveMQ](http://www.java-tv.com/2009/07/24/introduction-to-messaging-with-apache-activemq/) at [Øredev 2008 conference](https://oredev.org/) _by_ _[Bruce Snyder](http://bsnyderblog.blogspot.com/)_
+### Camel Videos(Lang: EN(GLISH), FRENCH, Norwegian, Russian, Spannish)
```

Comment:
```suggestion
### Camel Videos
```

Reply:
The videos you mention above are active and on the current list,